pkg/quota/v1/resources_test.go

```diff
@@ -0,0 +1,321 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package quota
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func TestEquals(t *testing.T) {
```

Comment:
I'm surprised this is a copy, not a move.  Is there an original somewhere?  Looks like the file it's testing has completely moved.

Reply:
yeah. am surprised too. 😅originally i didn't try to remove the old quota pkg. but after i finished switching controller part, the entire `pkg/quota` can vanish with a `rm -rf`.